Dockerfile

```diff
@@ -1,21 +1,22 @@
-FROM ubuntu:18.04
+FROM ubuntu:22.04
```

Comment:
If we're updating to 22.04 here, we should probably be doing so in the `Vagrantfile` too